tests/dummy/app/templates/application.hbs

```diff
@@ -133,11 +133,32 @@
     {{github-issue title="Focus Ordering"}}
 
     <h3>Query Params</h3>
-    <p>Scenarios to write:</p>
-    <ul>
-      <li>Refreshing model</li>
-      <li>Non-refreshing model</li>
-    </ul>
+    <p>These tests are for focusing behavior. You may use your mouse to click through the links in this test.</p>
+    <ol>
+      <li><button {{action "resetDemo"}}>Reset Demo</button></li>
+      <li>{{#link-to "feed"}}Load the Feed route.{{/link-to}}</li>
+      <li>Focus should be set to a block which begins with the "Feed" header.</li>
+
+      <li>{{#link-to "feed" (query-params sort="title")}}Update query params to sort by title.{{/link-to}}</li>
+      <li>Model was <em>not</em> refreshed; full transition <em>not</em> performed.</li>
+      <li>Items should reorder into alphabetical order.</li>
+      <li>No focus should be set.</li>
```

Comment:
@BrianSipple I'm pretty sure I agree with you and that the way you specified this is the "correct" behavior.

Refreshing the model implies new content, and a new page. The use case for this sort of behavior is passing an argument into some outbound query, something like search results.

The other type of query param–without refreshing the model–is for serializing render state into a query param. There should be some sort of communication for this behavior, but I don't believe that it belongs as part of outlet focusing–probably in the future "stream-hijacking" thing I want to build.

@Robdel12 et al. I'll still be interested in your opinion on the matter. :)